packages/strapi-plugin-users-permissions/services/User.js

```diff
@@ -48,6 +48,9 @@ module.exports = {
 
     // Use Content Manager business logic to handle relation.
     if (strapi.plugins['content-manager']) {
+      params.model = 'user';
+      params.id = params._id;
```

Comment:
`params.id = params._id || params.id`? Not sure but if the `params.id` wasn't null, it will override the value with the `undefined` value.